thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -786,6 +788,8 @@ export default Controller.extend({
     const { anomalyUrn, anomalyFeedback, sessionText } =
       this.getProperties('anomalyUrn', 'anomalyFeedback', 'sessionText');
 
+    if (!anomalyUrn) { return; }
+
     // debounce: do not run if destroyed
     if (this.isDestroyed) { return; }
```

Comment:
this vs line 764 `if (this.isDestroyed)`?

Reply:
hmm?

Reply:
maybe dup. We can remove one?

Reply:
why would this be a dup? sometime rca loads without an anomaly id, bt controller is not destroyed either

Reply:
resolved offline